templates/admin/base.html

```diff
@@ -1,7 +1,19 @@
 {% extends "admin/base.html" %}
 {% load static %}
 
+{% block extrastyle %}
+{{ block.super }}
+  <link href="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css" rel="stylesheet" />
+{% endblock extrastyle %}
+
 {% block extrahead %}
-  <script type="text/javascript"
-          src="{% static 'js/admin/extendFunctionality.js' %}"></script>
+{{ block.super }}
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+  <script>window.jQuery || document.write('<script src="js/jquery-3.6.0.min.js" type="text/javascript"><\/script>')</script>
+  <script type="text/javascript" src="{% static 'admin/author_slug.js' %}"></script>
+  <script type="text/javascript" src="{% static 'admin/play.js' %}"></script>
```

Comment:
Используй сокращённые теги
```suggestion
  <script type="text/javascript" src="{% static 'admin/author_slug.js' %}" />
  <script type="text/javascript" src="{% static 'admin/play.js' %}" />
```

Reply:
@AntonZelinsky что-то у меня такие сокращенные теги подсвечиваются как не верные. Если не трудно скиньте ссылку на подобные варианты, я чтото не могу найти (((